README.md

```diff
@@ -65,7 +65,12 @@ The following configuration options have defaults, so the configuration wizard d
 * the pull branch strategy
   * how to sync the main branch / perennial branches with their upstream
   * default: `rebase`
-  * possible values: `merge`, `rebase`
+  * possible values: `rebase`, `merge`
+
+* the git-hack push strategy
```

Comment:
Maybe `flag` instead of `strategy` since its not how we push but whether or not we do